cpp/arcticdb/column_store/memory_segment_impl.hpp

```diff
@@ -965,7 +965,7 @@ class SegmentInMemoryImpl {
                 using ColumnTagType = typename TypeDescriptorTag::DataTypeTag;
                 using RawType = typename ColumnTagType::raw_type;
 
-                util::BitSet final_bitset = std::move(filter_bitset);
```

Comment:
This move makes me wonder if the param was supposed to be lvalue instead of const ref.

Reply:
Yes, I have fixed this once somewhere. Maybe it's stuck on a branch. I'll dig it out.

Please don't merge this as is since it will cover up the genuine issue that the bitset is being copied by mistake.

Reply:
Would you mind porting the internal change to ArcticDB? Otherwise, we will lose it when we switch.